src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminal.accessibility.contribution.ts

```diff
@@ -53,6 +53,10 @@ class AccessibleBufferContribution extends DisposableStore implements ITerminalC
 	async createCommandQuickPick(): Promise<IQuickPick<IQuickPickItem> | undefined> {
 		return this._accessibleBufferWidget?.createQuickPick();
 	}
+
+	navigateToCommand(type: 'next' | 'previous'): void {
+		return this._accessibleBufferWidget?.navigateToCommand(type);
+	}
```

Comment:
A const enum with items assigned to numbers 1 and 2 would result in slightly less bytes in the final bundle